lib/elixir/lib/system.ex

```diff
@@ -331,6 +339,7 @@ defmodule System do
 
   The function must receive the exit status code as an argument.
   """
+  @spec at_exit((non_neg_integer -> any)) :: list(fun)
```

Comment:
I think we should return `:ok` here. The fact it returns `list(fun)` is an accident.